src/lib/chips/chip-list.ts

```diff
@@ -514,6 +530,20 @@ export class MatChipList implements MatFormFieldControl<any>, ControlValueAccess
     this._lastDestroyedIndex = null;
   }
 
+  /** Re-evaluates the error state. This is only relevant with @angular/forms. */
+  protected _updateErrorState() {
```

Comment:
This looks like the same logic as select and input- should it be a method in core? Or perhaps a mixin?